spec/lib/dice/die_cup_spec.rb

```diff
@@ -1,45 +1,40 @@
 # frozen_string_literal: true
 
-require_relative('../../../lib/dice/die')
-require_relative('../../../lib/dice/die_cup')
+require_relative '../../../lib/dice/die'
+require_relative '../../../lib/dice/die_cup'
 
-describe 'DieCup' do
-  it 'roll_range' do
-    die_cup = DieCup.new
-    die_cup.add_die(Die.new(6))
-    die_cup.add_die(Die.new(6))
+describe DieCup do
```

Comment:
- refactored the existing tests to be more readable
- added new tests for `Die` to reflect the context of arguments vs. no arguments received